.gitignore

```diff
@@ -34,7 +34,6 @@ nb-configuration.xml
 *.rej
 
 # Local environment
-.env
```

Comment:
why? do we need some secrets? then it might make sense creating an additional secrets.env and add this one to gitignore

Reply:
i did remove .env from gitignore, not add it
not sure why it was in there, i don't think we use it anywhere else